package-lock.json

```diff
@@ -0,0 +1,10937 @@
+{
```

Comment:
I thought you've already installed the `yarn`. Why did you execute the `npm install`?
I believe we can remove `package-lock.json` and go with `yarn.lock`